pkg/es/config/config.go

```diff
@@ -33,48 +33,60 @@ import (
 	storageMetrics "github.com/jaegertracing/jaeger/storage/spanstore/metrics"
 )
 
+type TemplateOptions struct {
+	Priority    int `mapstructure:"priority"`
+	NumShards   int `mapstructure:"num_shards"`
+	NumReplicas int `mapstructure:"num_replicas"`
+}
```

Comment:
Something changed since the last review - you decided to flatten template fields? Then is this struct still needed?

Reply:
we need it, we would be duplicating four different index Settings in Indices.